From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Wed, 22 Jul 2026 16:00:32 -0600
Subject: [PATCH] kernelbase: HACK: Force disable direct composition for Elisa:
 The Innkeeper - Prequel.

CW-Bug-Id: #27413
Valve-commit: 91642a856d573ad70ee07c7df00fe06a6966ffa0
Picked: clean
Class: valve
Evidence: docs/evidence/2026-10-05-deps-latest.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index b53abdc..f47bcfe 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -512,6 +512,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
     }
     options[] =
     {
+        {L"Elisa The Innkeeper - Prequel\\nw.exe", L" --disable_direct_composition=1"},
         {L"UniverseGenerator.exe", L" --disable_direct_composition=1"},
         {L"Blaite\\nw.exe", L" --disable_direct_composition=1"},
         {L"Click&Fight.exe", L" --disable_direct_composition=1"},
